### PR TITLE
Add Scrutinizer Build integration

### LINKED ISSUE #312: Add Scrutinizer Build
Scrutinizer now provides first-class support for builds and deployment; any chance of adding this build status as well as Scrutinizer's code coverage and code quality badges?

### PR DESCRIPTION
**Pull request by kyrias**
Closes #312

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1570,6 +1570,52 @@ cache(function(data, match, sendBadge, request) {
...
+  if ((repoParts.length - 1) > slashesInRepo) {
+    branch = repoParts[repoParts.length - 1];
+    repo = repoParts.slice(0, -1).join('/');
+  }
+  var apiUrl = 'https://scrutinizer-ci.com/api/repositories/' + repo;
+  var badgeData = getBadgeData('build', data);
+  request(apiUrl, {}, function(err, res, buffer) {
+    if (err !== null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
```
Comment (espadrine) [inline server.js]: Could you return here?

**Comment by espadrine** [comment]
Thanks!

**Comment by kyrias** [comment]
Heh, seems you merged this before I had time to add a return. Want me to do that and open a new PR or will you do it yourself?

**Comment by espadrine** [comment]
No, the rest was really good, so I figured that nitpick wasn't a big deal, and I added a commit to use a return ☺

**Pull request merged**